shared/nav.native.js

```diff
@@ -207,4 +208,17 @@ const mapDispatchToProps = (dispatch: Dispatch, ownProps: OwnProps) => ({
   },
 })
 
-export default connect(mapStateToProps, mapDispatchToProps)(Nav)
+const dismissKeyboardOnPathChange = lifecycle({
```

Comment:
just a fyi: i'm making a branch on top of this that adds more to this so i just moved it into the compose. it does more stuff than on 'OnPathChange' so i removed that part